resources/views/components/header.blade.php

```diff
@@ -1,27 +1,27 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-success">
     <div class="container">
-        <a class="navbar-brand" href="/">Warnetku</a>
+        <a class="navbar-brand" href="route('home')">Warnetku</a>
```

Comment:
Why this one is not inside ```{{ }}```?

Reply:
sorry seem like i forgot to add it, i am going to fix now.